cmd/explorer/main.go

```diff
@@ -509,7 +509,6 @@ func main() {
 			router.HandleFunc("/dashboard/save", handlers.UserDashboardWatchlistAdd).Methods("POST")
 
 			router.HandleFunc("/dashboard/data/allbalances", handlers.DashboardDataBalanceCombined).Methods("GET")
-			router.HandleFunc("/dashboard/data/balance", handlers.DashboardDataBalance).Methods("GET")
```

Comment:
I don't see this being used. The ones for the API are somewhere else, see

- apiV1Router.HandleFunc("/dashboard/data/balance",...
- apiV1Router.HandleFunc("/dashboard/data/balances",